pipeline/data_merging/aggregate_across_columns.py

```diff
@@ -209,27 +213,32 @@ def selectMaxAlleleFrequency(newRow):
                 maxFreq = aa
                 maxFreqString = "%f (AA from ESP)" % aa
     if newRow["EUR_Allele_frequency_1000_Genomes"] != EMPTY:
-        eur_af = float(newRow["EUR_Allele_frequency_1000_Genomes"])
+        freqs = [float(i) for i in newRow["EUR_Allele_frequency_1000_Genomes"].split(',')]
```

Comment:
Yes.  We'll have to address that (and other fields that could have atomic values or could be lists) when we have users wanting to query against these fields...